notebooks/llm-agent-functioncall/llm-agent-functioncall-qwen.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>could you please provide short description for each added tool?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I think the the 'description ' parameter in each tool class has already provided these messages ?

Reply:
I mean in markdown before cell, not all people carefully read code, it will allow bring more attention